pkg/noderesourcetopology/filter.go

```diff
@@ -198,12 +198,10 @@ func (tm *TopologyMatch) Filter(ctx context.Context, cycleState *framework.Cycle
 	}
 
 	nodeName := nodeInfo.Node().Name
-	nodeTopology, ok := tm.nrtCache.GetCachedNRTCopy(nodeName, pod)
-	if !ok {
-		return framework.NewStatus(framework.Unschedulable, fmt.Sprintf("invalid node topology data for node %s", nodeName))
-	}
-	if nodeTopology == nil {
-		return nil
+	nodeTopology, err := tm.nrtCache.GetCachedNRTCopy(nodeName, pod)
+	if err != nil {
+		klog.Error(err)
+		return framework.NewStatus(framework.Unschedulable, err.Error())
```

Comment:
why the other check was removed?

Reply:
In my first implementation I removed possibility of returning `nil,nil` from GetCachedNRTCopy, with next patchset I am going to revert it so will add this check again:
```go
if nodeTopology == nil {
	return nil
}
```